snuba/settings/__init__.py

```diff
@@ -6,7 +6,7 @@
 from snuba.datasets.partitioning import SENTRY_LOGICAL_PARTITIONS
 from snuba.settings.validation import validate_settings
 
-LOG_LEVEL = os.environ.get("LOG_LEVEL", "INFO")
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "WARN")
```

Comment:
Hold on. This way we will loose logs we actually rely on today (unless we start those pods explicit as info):
- Batches commit in the consumers 
- Partitions assigned/revoked 

I don't think we should loose those

Reply:
Ah, fair enough. I assumed this was going to be a more straightforward task